k-distribution/tests/regression-new/markdownSelectors/test.md

```diff
@@ -28,7 +28,7 @@ module TEST
   rule <k> 1 => 2 </k>
 ```
 
-```{.k .discard .numberLines startFrom="0"}
```

Comment:
Please keep it here as it tries the new syntax.

Reply:
Okay, I will revert all changes to markdown files.

Reply:
No, I meant only in this `test.md` file

Reply:
Ah okay. I will make the changes.